quinn-proto/src/connection/mod.rs

```diff
@@ -889,15 +889,15 @@ where
     pub fn read_unordered(&mut self, id: StreamId) -> Result<Option<(Bytes, u64)>, ReadError> {
         Ok(self.streams.read_unordered(id)?.map(|result| {
             self.add_read_credits(id, result.max_stream_data, result.max_data);
-            (result.buf, result.offset)
+            (result.result.0, result.result.1)
```

Comment:
Desctructuring `result` into `bytes` and `offset` would make this line a bit clearer.

Reply:
I ended up changing it to just returning `result.result`, as in the other methods. In that case, just naming the values in it feels a little superfluous, since they immediately get repacked into a tuple again.